crates/wasm-encoder/src/aliases.rs

```diff
@@ -0,0 +1,63 @@
+use super::*;
+
+/// An encoder for the alias section.
+pub struct AliasSection {
```

Comment:
All of the other sections have an example in the doc comment, could you add one here as well? Also please mention that this section is part of the module linking proposal.